Cargo.toml

```diff
@@ -75,6 +75,7 @@ structopt = "0.3"
 tokio = { version = "0.2.22", features = ["uds"] }
 tokio-util = { version = "0.3", features = ["codec"] }
 url = "2.1.1"
+flexbuffers = "2.0"
```

Comment:
Please see my comment on the contents of `async fn forward_gsb`

Reply:
removed